server/src/tree-sitter/analyzer.ts

```diff
@@ -790,14 +817,29 @@ export default class Analyzer {
     /**
      TODO: Achieve the following goals using the scan results
        [check] 1. Get the final values for variables and show them in the hover
-       2. Get the complete history of assignments for variables and use them in the go-to-definition feature
+       [check] 2. Get the complete history of assignments for variables and use them in the go-to-definition feature
```

Comment:
I don't see this taking effect on the definitions. I started the scan recipe command, then got the same results:
![image](https://github.com/yoctoproject/vscode-bitbake/assets/19970889/ae088e11-43c7-46e9-9077-b5bc273ffbf5)
Here, the nonexistent override should not be reported any longer, and a weak assignment I put in builc/conf/local.conf should also appear.

Reply:
Ok that's the proper result then if it's reported in bitbake -e. thanks for the clarification!